### ISSUE TITLE
Error removing old snapshots

### ISSUE THREAD

**Issue by toketin** [restic/restic#3606]
Hi, I've tried to remove old snapshots of my restic backup using the command on the manual (`sudo restic -r /srv/restic-repo forget --keep-daily 4 --prune`), but I've got this output:
```
4 snapshots have been removed, running prune
counting files in repo
building new index for repo
[0:09] 100.00%  780 / 780 packs
repository contains 780 packs (103808 blobs) with 3.352 GiB
processed 103808 blobs: 0 duplicate blobs, 0 B duplicate
load all snapshots
find data that is still in use for 4 snapshots
[0:49] 100.00%  4 / 4 snapshots
Fatal: number of used blobs is larger than number of available blobs!
Please report this error (along with the output of the 'prune' run) at
https://github.com/restic/restic/issues/new
```
I'm running Restic version 0.9.4 on Raspbian.

What could I do in order to avoid this error?

Thanks in advance!
Marco

**Comment by MichaelEischer** [comment]
The repository seems to be damaged in some way. Please upgrade to a recent restic version, then run `restic check --read-data` to get more information.

**Comment by toketin** [comment]
Thank you Michael, I've upgraded using the pre-compiled binaries of restic, since on Raspbian stable repo up to now there's just the 0.9.4 version. Then I've created again the backup since the repo was damaged. Rebuilding again the index after the prune hasn't fixed the issue.

**Comment by MichaelEischer** [comment]
> Then I've created again the backup since the repo was damaged.

I'm not sure I understand. Did you run `restic backup` again or create a new repository?

> Rebuilding again the index after the prune hasn't fixed the issue.

Please provide the information I've asked for above. There's no point in trying to guess what the problem is.

**Comment by toketin** [comment]
Yes I've deleted tha restic repo and I've created it again. The trouble was that after creating a new snapshot and running the command `sudo restic -r /srv/restic-repo forget --keep-daily 4 --prune` again restic told that the index was damaged. 

I've tried to rebuild it again and creating a new snapshot, but nothing restic told that were missing many files, so I took the chance to delete all the restic repo dir and build it as new.
